src/adcm_client/objects.py

```diff
@@ -1444,6 +1445,194 @@ class ConcernList(BaseAPIListObject):
+    id = None
+    username = None
+    first_name = None
+    last_name = None
+    email = None
+    is_superuser = None
+    password = None
+    profile = None
+
+    def group(self) -> "GroupList":
+        """Return list of `Group` object"""
+        # TODO: I can't do this, because search() is not working
+        # return GroupList(self._api, user=self.id)
+        groups = GroupList(self._api)
+        data = []
+        for group in self._data['group']:
+            data.append(Group(self._api, id=group['id']))
+        groups.data = data
+        return groups
+
+    def change_password(self, password: str) -> None:
+        """Changing user password"""
+        self._api.objects.rbac.user.partial_update(id=self.id, password=password)
+        self.reread()
+
+
+class UserList(BaseAPIListObject):
+    """List of `User` objects"""
+
+    _ENTRY_CLASS = User
+
+
+@allure_step('Create user {username}')
+def new_user(api: ADCMApiWrapper, username: str, password: str, **kwargs):
+    """Create new `User` object"""
+    user = api.objects.rbac.user.create(
+        username=username, password=password, **strip_none_keys(kwargs)
+    )
+    return User(api, id=user['id'])
+
+
+##################################################
+#              G R O U P
+##################################################
+class Group(BaseAPIObject):
+    IDNAME = 'id'
+    PATH = ['rbac', 'group']
+    FILTERS = ['id', 'name', 'group']
+    id = None
+    name = None
+    description = None
+
+    def user(self) -> "UserList":
+        # TODO: I can't do this, because search() is not working
+        # return UserList(self._api, group=self.id)
+        users = UserList(self._api)
+        data = []
+        for user in self._data['user']:
+            data.append(User(self._api, id=user['id']))
+        users.data = data
+        return users
+
+    def add_user(self, user: User) -> None:
+        """Adding a user to a group"""
+        current_users = self.user()
+        users = [{'id': obj.id} for obj in current_users]
+        users.append({'id': user.id})
+        self._api.objects.rbac.group.partial_update(id=self.id, user=users)
+        user.reread()
+        self.reread()
+
+
+class GroupList(BaseAPIListObject):
+    _ENTRY_CLASS = Group
+
+
+@allure_step('Create group {name}')
+def new_group(api: ADCMApiWrapper, name: str, **kwargs):
+    """Create new `Group` object"""
+    group = api.objects.rbac.group.create(name=name, **strip_none_keys(kwargs))
+    return Group(api, id=group['id'])
+
+
+##################################################
+#              R O L E
+##################################################
+class Role(BaseAPIObject):
+    IDNAME = 'id'
+    PATH = ['rbac', 'role']
+    FILTERS = ['id', 'name', 'display_name', 'built_in', 'type', 'child']
+    id = None
+    name = None
+    description = None
+    built_in = None
+    type = None
+    category = None
+    parametrized_by = None
+
+    def child(self) -> "RoleList":
```

Comment:
I am in doubt about signature. I prefer to have *_list name for any function, which returns some like list

Reply:
fixed